### PR TITLE
en: add `clean-dump-file` configuration

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

add configuration introduced by pingcap/dm#804

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/170, https://github.com/pingcap/docs-dm/pull/169
- Other reference link(s):

### CODE REVIEW

**Comment by toutdesuite** [comment]
@lance6716 PTAL, thanks~

**Comment by lance6716** [comment]
LGTM

File: en/task-configuration-file-full.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ heartbeat-report-interval: 10   # The interval at which DM estimates the lag (th
 timezone: "Asia/Shanghai"       # The timezone.
 case-sensitive: false           # Determines whether the schema/table is case-sensitive.
 online-ddl-scheme: "gh-ost"     # Only "gh-ost" and "pt" are currently supported.
+clean-dump-file: true           # Whether to clean up the files generated during data dump. Note that these include `metadata` files.
```
Comment (TomShawn) [inline en/task-configuration-file-full.md:38]: @toutdesuite The upstream in zh is `是否清理 dump 阶段产生的文件，包括 metadata 文件、建库建表 SQL 文件以及数据导入 SQL 文件`. Looks like something is missing.

Comment (toutdesuite) [reply en/task-configuration-file-full.md:38]: Confirmed with the author. `建库建表 SQL 文件以及数据导入 SQL 文件` can be ignored~

Comment (lance6716) [reply en/task-configuration-file-full.md:38]: found it a bit hard to find proper translation for "建库建表 SQL 文件以及数据导入 SQL 文件", so we choose to omit it 😂

Comment (lance6716) [reply en/task-configuration-file-full.md:38]: the original purpose to mention "metadata 文件、建库建表 SQL 文件以及数据导入 SQL 文件" is only to stress `metadata` file, so only address it in English version

**Pull request merged**